rocAL/rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -131,11 +131,23 @@ extern "C" void ROCAL_API_CALL rocalGetImageLabels(RocalContext rocal_context, v
 /// \return The size of the buffer needs to be provided by user to get bounding box info for all images in the output batch.
 extern "C" unsigned ROCAL_API_CALL rocalGetBoundingBoxCount(RocalContext rocal_context, int* buf);
 
+///
+/// \param rocal_context
+/// \param buf the imageIdx in the output batch
+/// \return The size of the buffer needs to be provided by user to get mask box info associated with image_idx in the output batch.
+extern "C" unsigned ROCAL_API_CALL rocalGetMaskCount(RocalContext rocal_context, int* buf );
+
+///
+/// \param rocal_context
+/// \param bufcount The user's buffer that will be filled with polygon size for the mask info
+/// \param buf The user's buffer that will be filled with mask info for the images in the output batch. It needs to be of size returned by a call to the rocalGetMaskCount
+extern "C" void ROCAL_API_CALL rocalGetMaskCoordinates(RocalContext rocal_context, int* bufcount, float* buf);
```

Comment:
I think mask_count can be extracted from this api. In that case do you need the api on #138

Reply:
rocalGetMaskCount - returns the number of polygons for each bbox annotation for an image
rocalGetMaskCoordinates - returns the number of mask values in each polygon for each bbox annotation for an image